source/features/conflict-marker.tsx

```diff
@@ -0,0 +1,90 @@
+import select from 'select-dom';
+import React from 'dom-chef';
+import * as api from '../libs/api';
+import {getOwnerAndRepo, getRepoURL} from '../libs/utils';
+import features from '../libs/features';
+import {alert} from '../libs/icons';
+
+interface PRConfig {
+	id: string;
+	owner: string;
+	name: string;
+}
+
+function getPrNumber(pr: string): string {
+	return pr.split('_')[1];
+}
+
+function createQueryFragment(pr: PRConfig): string {
+	return `
+		${api.escapeKey(pr.id)}: repository(owner: "${pr.owner}", name: "${pr.name}") {
+			pullRequest(number: ${getPrNumber(pr.id)}) {
+				mergeable
+			}
+		}
+	`;
+}
+
+function buildQuery(
+	prs: PRConfig[]
+): string {
+	return prs.map(createQueryFragment).join('\n');
+}
+
+function getPRConfig(element: HTMLElement): PRConfig {
+	try {
+		const prTitle = select('a[data-hovercard-type="repository"]', element)!.textContent!;
+		const [owner, name] = prTitle.trim().split('/');
+
+		return {
+			id: element.id,
+			owner,
+			name
+		};
+	} catch (error) {
+		const {ownerName, repoName} = getOwnerAndRepo();
+
+		return {
+			id: element.id,
+			owner: ownerName,
+			name: repoName
+		};
+	}
+}
+
+async function init(): Promise<false | void> {
+	const elements = select.all('.js-issue-row');
+	if (elements.length === 0) {
+		return false;
+	}
+
+	const prs = elements.map(getPRConfig);
+
+	const query = buildQuery(prs);
+	const data = await api.v4(query);
+
+	for (const pr of elements) {
+		if (data[api.escapeKey(pr.id)].pullRequest.mergeable === 'CONFLICTING') {
+			select('.d-inline-block.mr-1', pr)!.insertAdjacentElement('afterbegin',
```

Comment:
Since we're dealing with links, you can just add it after:

```suggestion
			prLink.after(
```

Reply:
> Side note: Always prefer the newer DOM APIs like before after prepend and append. afterbegin was the same as prepend.

@fregante => https://github.com/sindresorhus/eslint-plugin-unicorn/issues/new ;)

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/issues/344